communities/ucf-new-to-research/community-assessment.md

```diff
@@ -0,0 +1,45 @@
+# Community Assessment: Undergraduate students conducting Computer Science research at UCF
+
+## Definition
+* Undergraduate students either interested in conducting or already conducting Computer Science research at the University of Central Florida.
+
+## Size
+* 140 members of Mad Scientists Research Society (MSRS) Facebook group (the only student-led organization for undergraduate research at UCF); since the group is not specific to Computer Science, the actual number is likely much smaller.
+* No information on number of students overall that are involved/interested in CS research.
+* Would like to have between 200-300 people in the community given the size of the school, or, as many as possible.
+
+## Resources
+* MSRS has limited funding. The group runs a mentorship program for junior/seniors to assist freshman and sophomores. The club does hold meetings, but infrequently. The club also runs a day of student research talks each semester. More experienced students in the organization are resources for less experienced students.
```

Comment:
Oh wow, that's really cool. I'd love to hear more about how the mentorship program works